protected/models/cron/cron.object.php

```diff
@@ -38,42 +20,31 @@ public function create(array $values = []) : bool
     }
 
     /**
-     * Update Cron Job Data In Data Base By Job ID
-     *
-     * @param array $columns Cron Job Columns In Data Base Table
-     * @param array $values  Cron Job Data
-     * @param int   $id      Cron Job ID
-     *
-     * @return bool Is Cron Job Data Successfully Updated
+     * summary
      */
     public function updateCronByID(
-        array  $columns = [],
-        array  $values  = [],
-        int    $id      = -1
-    ) : bool
+        ?array  $columns = null,
```

Comment:
Codacy found an issue: [TRUE, FALSE and NULL must be uppercase; expected "NULL" but found "null"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)